tox-windows.ini

```diff
@@ -0,0 +1,94 @@
+[tox]
```

Comment:
Do we need another `tox.ini` just for windows? We should be able to add this stuff into the existing `tox.ini`. Running tests in different environments is what `tox` is made to do.

Reply:
Unless there's a compelling reason to keep them separate, they should be merged as part of this PR.

Reply:
doing a `tox -e ALL` will not work as it will run both platforms.

Reply:
We don't need to run all environments by default. You can configure tox to run only builds that'll succeed on Linux by default, and Windows environments only if specifically requested. https://tox.readthedocs.io/en/latest/config.html#confval-envlist=CSV

Reply:
OK, I have merged the two by just using the 'platform=Linux/win32'

Reply:
Fixed.